wordcloud/views.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+from django.http import HttpResponse
+from django.utils import simplejson
+from django.conf import settings
+
+from .wordcloud import popular_words
+
+
+def wordcloud(request, max_entries=30):
```

Comment:
It might be worth adding a Cache-Control header to increase the time the wordcloud data is cached for (e.g. to a few hours)